l10n_it_vat_registries/wizard/print_registro_iva.py

```diff
@@ -0,0 +1,167 @@
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import _, api, fields, models
+from odoo.exceptions import UserError
+
+
+class WizardRegistroIva(models.TransientModel):
+    _name = "wizard.registro.iva"
+    _description = "Run VAT registry"
+
+    date_range_id = fields.Many2one("date.range", string="Date range")
+    from_date = fields.Date("From date", required=True)
+    to_date = fields.Date("To date", required=True)
+    layout_type = fields.Selection(
+        [
+            ("customer", "Customer Invoices"),
+            ("supplier", "Supplier Invoices"),
+            ("corrispettivi", "Sums due"),
+        ],
+        "Layout",
+        required=True,
+        default="customer",
+    )
+    show_full_contact_addess = fields.Boolean()
+    tax_registry_id = fields.Many2one("account.tax.registry", "VAT registry")
+    entry_order = fields.Selection(
+        [
+            ("date_name", "Date - Number"),
+            ("journal_date_name", "Journal - Date - Number"),
+        ],
+        default="date_name",
+        required=True,
+    )
+    journal_ids = fields.Many2many(
+        "account.journal",
+        "registro_iva_journals_rel",
+        "journal_id",
+        "registro_id",
+        string="Journals",
+        help="Select journals you want retrieve documents from",
+    )
+    include_rc_moves = fields.Boolean(
+        string="Include reverse charge moves",
+    )
+    rc_journal_ids = fields.Many2many(
+        "account.journal",
+        "registro_iva_rc_journals_rel",
+        "journal_id",
+        "registro_id",
+        string="Reverse charge journals",
+        help="Select journals you want retrieve reverse charge documents from",
+    )
```

Comment:
```suggestion
    rc_journal_ids = fields.Many2many(
        "account.journal",
        "registro_iva_rc_journals_rel",
        "registro_id",
        "journal_id",
        string="Reverse charge journals",
        help="Select journals you want retrieve reverse charge documents from",
    )
```
Se poi potessimo anche usare l'inglese sarebbe meglio :smile:, tipo `registry_id` invece di `registro_id`.

Reply:
Cioè invertiresti l'ordine ? La prima colonna non punto al record di `account.journal`?

Reply:
> Cioè invertiresti l'ordine ?

Sì perchè i `Many2Many` sono definiti così, vedi https://github.com/odoo/odoo/blob/44e61c6b429aab94e4656ade5e0f0275b905210a/odoo/fields.py#L4846-L4850:
>    :param str column1: optional name of the column referring to "these" records
>        in the table ``relation``
>
>    :param str column2: optional name of the column referring to "those" records
>        in the table ``relation``

In questo caso "these" sono i registri IVA e "those" sono i registri di Odoo.

Reply:
fatto